treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -48,32 +48,31 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity, pa
 
     repository = Repository.objects.get(name=project_name)
 
+    defaults = {
+        'test': signature_props.get('test', ''),
+        'suite': signature_props['suite'],
+        'option_collection': option_collection,
+        'platform': platform,
+        'framework': framework,
+        'extra_properties': extra_properties,
+        'has_subtests': signature_props.get('has_subtests', False),
```

Comment:
By default parent_signature will be null if not's set, so that's fine. With has_subtests, it's better to set it explciitly, so this code is good as-is, I think.